src/common/errors.js

```diff
@@ -1,7 +1,18 @@
 'use strict';
 const util = require('util');
 
-class RippleError extends Error {
+// this is needed because extending builtins doesn't work in babel 6.x
+function extendableBuiltin(cls) {
+  function ExtendableBuiltin() {
+    cls.apply(this, arguments);
+  }
+  ExtendableBuiltin.prototype = Object.create(cls.prototype);
+  Object.setPrototypeOf(ExtendableBuiltin, cls);
+
+  return ExtendableBuiltin;
+}
+
+class RippleError extends extendableBuiltin(Error) {
```

Comment:
@lumberj it's not thing that I've found first
https://phabricator.babeljs.io/T3083
in short - subclassing builtins wasn't supported in Babel 5 either, it just behaved differently